winforms/src/toga_winforms/widgets/base.py

```diff
@@ -49,7 +85,8 @@ def __init__(self, interface):
         self._container = None
         self.native = None
         self.create()
-        self.init_scale(self.native)
+        # Required to prevent Hwnd Related Bugs
+        self.native.CreateGraphics().Dispose()
```

Comment:
We'll need a more detailed comment than that.

Reply:
Done. Thanks!